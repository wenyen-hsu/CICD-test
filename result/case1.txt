Voltage: 1.27V
Delay = 0.92ns
Power: 0.305mW
Temperature: 37.6C
Leakage: 0.0030mA
Frequency: 1.18GHz
Area: 1201.2um2
Efficiency: 97.0%