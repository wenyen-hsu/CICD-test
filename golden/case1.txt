Voltage: 1.25V
Delay = 0.89ns
Power: 0.31mW
Temperature: 37.5C
Leakage: 0.0025mA
Frequency: 1.20GHz
Area: 1200.0um2
Efficiency: 97.8%